components/radio-button/script.js

```diff
@@ -20,42 +20,70 @@ class GamefaceRadioGroup extends HTMLElement {
 	}
 
 	get value() {
+		if (this.disabled) return undefined;
+		if (this.previouslyCheckedElement && this.previouslyCheckedElement.disabled) return undefined;
 		if (this.previouslyCheckedElement) return this.previouslyCheckedElement.value;

```

Comment:
```suggestion
		if (this.disabled) return;
		if (this.previouslyCheckedElement && this.previouslyCheckedElement.disabled) return;
		if (this.previouslyCheckedElement) return this.previouslyCheckedElement.value;
```

There is no point in returning `undefined`, it will be returned anyway.

Reply:
You are totally right, fixed!